engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -479,10 +481,12 @@ public void startApplication() throws YarnException, IOException
         dag.setAttribute(LogicalPlan.CONTAINER_OPTS_CONFIGURATOR, new BasicContainerOptConfigurator());
       }
 
+      log4jPropFile = StramClientUtils.getConfigDir() + File.separator + APEX_LOG4J_PROPS_FILE;
```

Comment:
As mentioned before this won't work if user changes the value in -Dlog4j.configuration=log4j.props, correct?